elks/include/linuxmt/config.h

```diff
@@ -16,8 +16,13 @@
  * be overridden for embedded systems with less overhead.
  * See setup.S for more details.
  */
+#if defined(CONFIG_HW_HOMEBREW)
+#define SETUP_VID_COLS          int88(0)        /* BIOS video # columns */
+#define SETUP_VID_LINES         int88(1)        /* BIOS video # lines */
+#else
 #define SETUP_VID_COLS          setupb(7)       /* BIOS video # columns */
 #define SETUP_VID_LINES         setupb(14)      /* BIOS video # lines */
+#endif /* CONFIG_HW_HOMEBREW */
```

Comment:
I would prefer that this entire `#ifdef CONFIG_HW_HOMEBREW` section be moved past line 46, so that the standard CONFIG_ARCH_IBMPC settings remain together, even if it means having to use `#undef SETUP_VID_LINES` etc in that section.

(Note also we prefer `#ifdef CONFIG_xxx` rather than `#if defined(CONFIG_xxx)`. You're welcome to fix the `#if defined(CONFIG_MK_88)` below as well - I missed that when it was submitted).